### PR TITLE
Adding secrets as flex volumes

### PR DESCRIPTION
**Pull request by adusumillipraveen**
Adding secrets as flex volumes 
Upgrading NodeJs Chart

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-1005

### Change description ###

Adding secrets as flex volumes 
Upgrading NodeJs Chart

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@hmcts/nodejs-healthcheck": "^1.4.5",
     "@hmcts/nodejs-logging": "^3.0.0",
+    "@hmcts/properties-volume": "0.0.9",
```
[the remark this answers is not in the document]
Comment (adusumillipraveen) [reply package.json]: yes, just waiting for travis to build new version. Was hoping it to be quick :)  The earlier version had a vulnerability.

File: app/service/service-token-generator.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const fetch = require('../util/fetch');
 
 const idamS2SUrl = config.get('idam.s2s_url');
 const serviceName = config.get('idam.service_name');
-const secret = config.get('idam.service_key');
+const secret = config.get('secrets.s2s.microservicekey-ccd-gw');
```
Comment (MSancaktutar) [inline app/service/service-token-generator.js]: Well, not a comment at the heart of the logic of the work, but miscroservicekey should be microservice-key. Or, do we really need to mention "microservice"? If this is something under s2s, the rest may omit microservice at all.

Comment (timja) [reply app/service/service-token-generator.js]: its coming from the s2s vault, its an exact match on the name of the key in vault

Comment (dwaynebailey) [reply app/service/service-token-generator.js]: This is just the name of the key in the vault.  In this case the `s2s` vault has a key called `microservicekey-ccd-gw`.  So not much of an option and this follows the names of all the s2s microservice keys in that vault

**Pull request merged**